waspc/src/Wasp/Generator/FileDraft/WriteableMonad.hs

```diff
@@ -7,7 +7,10 @@ import Control.Monad.IO.Class (MonadIO)
 import Data.Aeson as Aeson
 import Data.Text (Text)
 import qualified Data.Text.IO
+import qualified Path.IO as PathIO
 import StrongPath (Abs, Dir, File', Path', Rel)
+import qualified StrongPath.Path as SP.Path
+import StrongPath.Types (Dir')
```

Comment:
You can just add this to the import above `import StrongPath (Abs, Dir, File', Path', Rel)`. I guess it was auto import.